src/plugins/intel_cpu/src/utils/ngraph_transformation.hpp

```diff
@@ -67,11 +66,6 @@ class TransformationDumper {
 
         ov::pass::Manager serializer;
 
-        if (config.dumpIR.format.filter[DebugCapsConfig::IrFormatFilter::XmlBin]) {
-            auto custom_opsets = std::make_shared<Extension>()->getOpSets();
-            serializer.register_pass<ov::pass::Serialize>(pathAndName + ".xml", "", custom_opsets);
-        }
```

Comment:
Do we have other way to dump custom's ops?

Reply:
@v-Golubev Hi Golubev, can u help take a look this changes? ie::extension will be replaced by ov::extension, this API will be removed, is it fine for this:
https://github.com/openvinotoolkit/openvino/blob/9638b899db494ab7f8424191b15684f598a9f89a/src/plugins/intel_cpu/src/utils/ngraph_transformation.hpp#L71